examples/jsm/renderers/webgpu/WebGPUUniformBuffer.js

```diff
@@ -0,0 +1,55 @@
+import WebGPUBinding from './WebGPUBinding.js';
+import { GPUBindingType } from './constants.js';
+
+class WebGPUUniformBuffer extends WebGPUBinding {
+
+	constructor( name, buffer = null ) {
+
+		super( name );
+
+		this.onBeforeUpdate = function () {};
+
+		this.bytesPerElement = Float32Array.BYTES_PER_ELEMENT;
+		this.type = GPUBindingType.UniformBuffer;
+		this.visibility = GPUShaderStage.VERTEX;
+
+		this.usage = GPUBufferUsage.UNIFORM | GPUBufferUsage.COPY_DST;
+
+		this.buffer = buffer;
+		this.bufferGPU = null; // set by the renderer
+
+	}
+
+	setOnBeforeUpdate( callback ) {
```

Comment:
Do we still need this method? `onBeforeUpdate()` is still called in `WebGPUBindings` however it seems we never assign a callback.

Reply:
Yeah, I will remove.